alert/lib/script.py

```diff
@@ -27,7 +27,7 @@ def get_rep(self) -> str:
         return ",".join(self.value)
 
 
-def graph_of_grammar(grammartext):
+def graph_of_grammar(grammartext, filename=""):
```

Comment:
A better function signature would be `graph_of_grammar(grammertext, filename=None) -> str` and then if filename is None generate a uuid inside here and then return the filename at the end of the method. This would be more flexible as the caller does not need to know the filename if it doesn't need to.